metadata-service/restli-servlet-impl/src/main/resources/index/usage-event/aws_es_usage_event.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Quick question - why are we calling this "aws_es_usage_event"... is there something particularly AWS about it?

Reply:
Nvm - I see this is indeed configuration specific to elasticsearch